content/en/docs/Installation/pre-built-binaries.md

```diff
@@ -25,3 +25,119 @@ You can install Navidrome via the
 the [AUR](https://aur.archlinux.org/). Alternatively, you can install using the
 [`navidrome-git`](https://aur.archlinux.org/packages/navidrome-git/) package
 which will compile Navidrome from source.
+
+### Ubuntu Installation
+
+The following steps have been tested on Ubuntu 18.04 and should work on all version 16.04 and above as well as other Debian based distros. Throughout these instructions the commands will have placeholders for the user (`<user>`) and group (`<group>`) you want to run Navidrome under and the music folder path (`<library_path>`). If you are using an existing media library ensure the user has permissions to the media library.
+
+#### Update and Install Prerequisites
+
+Ensure your OS is up to date and install [ffmpeg](https://ffmpeg.org/download.html).
+
+```
+sudo apt update
+sudo apt upgrade
+sudo apt install vim ffmpeg
+```
+
+#### Create Directory Structure
+
+Create a directory to store the Navidrome executable and a working directory with the proper permissions.
+
+```
+sudo install -d -o <user> -g <group> /opt/navidrome
+sudo install -d -o <user> -g <group> /var/lib/navidrome
+```
+
+#### Get Navidrome
+
+Download the latest release from the [releases page](https://github.com/deluan/navidrome/releases), extract the contents to the executable directory, and set the permissions for the files.
+
+```
+wget https://github.com/deluan/navidrome/releases/download/v0.19.0/navidrome_0.19.0_Linux_x86_64.tar.gz -O Navidrome.tar.gz
+sudo tar -xvzf Navidrome.tar.gz -C /opt/navidrome/
+sudo chown -R <user>:<group> /opt/navidrome
+```
+
+#### Create Configuration File
+
+In the working directory, `/var/lib/navidrome` create a new file named `navidrome.toml` with the following settings.
+
+```
+LogLevel = "INFO"
+BaseURL = ""
+ScanInterval = "10s"
+TranscodingCacheSize = "15MB"
+MusicFolder = "<library_path>"
```

Comment:
I suggest keeping the config file as simple as possible, changing it to just:

```
MusicFolder = "<library_path>"
```

Reply:
Based on your unit example, don't we need to set the DataFolder to `/var/lib/navidrome`?

Reply:
NVM, just realized that you set the WorkingDirectory to it, should be fine, then 👍

Reply:
The minimal config makes sense, I update that section and change some wording in a few places